src/user-has-access-react.component.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { getCurrentUser } from "@openmrs/esm-api";
+
+export default function UserHasAccessReact(props) {
+  const [user, setUser] = React.useState(null);
+
+  function hasAccess(requiredPrivilege) {
```

Comment:
This function should be exported from openmrs-esm-api, requiring both a user and a requiredPrivilege as arguments.

Reason is for non-react use cases and for within-React-but-not-as-a-component use cases.

Reply:
Done

Reply:
👍 approved and merged https://github.com/openmrs/openmrs-esm-api/pull/6